docs/modules/ROOT/pages/how-to-guides/Import-code/proc_importing_code.adoc

```diff
@@ -123,9 +123,10 @@ In the **Build & deploy configuration** section, look over these fields:
 
 When you’re satisfied with your component configuration settings, click **Create application**. 
 
-[NOTE]
+[NOTES]
 ====
-After you create your application, you can adjust your configuration settings anytime you want from the **Application** view.
+* After you create your application, you can adjust your configuration settings anytime from the **Application** view.
+* From the **Components** tab, you can add a build or deployment secret to keep your data private. For more information about secrets, see link:https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_creating-secrets-for-your-builds/[Creating secrets for your builds].
```

Comment:
@jseseCCS We no longer have a way to add build or deployment secret from components tab. we now have a standalone page for managing secrets.

Secrets page:

![image](https://github.com/redhat-appstudio/docs.appstudio.io/assets/9964343/148852fa-ac82-4132-9787-7c792aacee47)

The hyperlink [Creating secrets for your builds](https://redhat-appstudio.github.io/docs.appstudio.io/Documentation/main/how-to-guides/configuring-builds/proc_creating-secrets-for-your-builds) content also needs to be updated to use the new Secret management page from the left navigation.

Reply:
man this has sat a while! i'll rework. thank you! don't go far. ;)

Reply:
@karthikjeeyar i made the nec. change. how is this? i'll change the "Creating secrets for your builds" content next.

Reply:
Thanks @jseseCCS. changes looks good to me